services/object/procs.yml

```diff
@@ -2,6 +2,11 @@ web:
   type: web
   command: node services/object/src/main.js server
   defaultReplicas: 0
+cleanup-expire-objects:
+  type: cron
+  schedule: '0 0 * * *'
+  deadline: 86400
+  command: node services/object/src/main cleanup-expire-objects
```

Comment:
Is there an easy way to make sure this doesn't run on production deployments, similar to the `defaultReplicas: 0` for the web proc?

Reply:
We can probably add an additional field `ignore` to ignore the cron command if `ignore` is set to true. Would you like me to add this?